mlflow/tracking/client.py

```diff
@@ -2738,6 +2748,11 @@ def print_model_version_info(mv):
             Version: 1
             Tags: {'t': '1'}
         """
+        _validate_model_version_or_stage_exists(version, stage)
+        if stage:
+            latest_versions = self.get_latest_versions(name, stages=[stage])
+            version = latest_versions[0].version if latest_versions else None
```

Comment:
How to handle the case `version` = None ?

Reply:
Wouldn't this catch above case ? https://github.com/mlflow/mlflow/blob/8693a26bf2acd418aa93aa5ea2e5536caa199fc3/mlflow/store/model_registry/sqlalchemy_store.py#L813